File: MODUL-2/Python/PRAK203-2310817220002-RaudatulSholehah.py
print("input")
Nilai = input().split()

if len(Nilai)==2:

    a, b = map(float, Nilai)
    i, j = map(float, input().split())
    x, y = map(float, input().split())

elif len(Nilai)==6:
    a, b, i, j, x, y = map(float,Nilai)

Hasil = ( a - b ) * ( i / j ) - ( x + y )
print("output")
print(f"{Hasil:.3f}")

File: MODUL-2/C/PRAK203-2310817220002-RaudatulSholehah.c
#include <stdio.h>
int main()
{
    float a, b, i, j, x, y, hasil;
    printf("Input\n");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    hasil = (a - b) * (i / j) - (x + y);

    printf("Output\n %.3f\n", hasil);
    return 0;
}